app-lets-encrypt/init.sls

```diff
@@ -0,0 +1,53 @@
+#
+# Ubiquitous Moodle
+#
+# @author Luke Carrier <luke@carrier.im>
+# @copyright 2016 Luke Carrier
+#
+
+include:
+  - app-base
```

Comment:
It still wouldn't help -- the issue then would be that neither `app-base` nor `nginx-base` would create the necessary configuration files. I think the tidiest solution to this would be moving the per-site configuration into a macro inside `app-lets-encrypt` that we call somewhere inside the `app-moodle` and `app-saml` roles.

Happy to leave this point for now and sit and work on this together at some point this week.